MeretzLib/Apple/iOS/Meretz/Meretz/Meretz.m

```diff
@@ -239,6 +258,39 @@ - (void) setMeretzUserAccessToken: (NSString *) accessToken
 		return;
 	}
 
```

Comment:
The new high-level APIs are just a thin wrapper around the low-level task management system. They spin up the same job, but discard the returned taskId handle (task is still tracked internally). The delegate methods are invoked upon task completion internally (see below).